lcpp.lua

```diff
@@ -281,13 +281,11 @@ end
 
 -- returns the number of string occurrences
 local function findn(input, what)
-	local count = 0
 	local offset = 0
 	local _
-	while true do 
+	for count=0,math.huge do

```

Comment:
what this change is for? it seems that this change practically no difference. if there is a bug like lcpp never break this loop, please report such a bug on issues.